server/src/application/application.service.ts

```diff
@@ -22,6 +22,18 @@ export class ApplicationService {
       });
     }
 
+    if (query.confirmationId) {
+      queryBuilder.andWhere('app.confirmationId ILIKE :confirmationId', {
+        confirmationId: `%${query.confirmationId}%`,
+      });
+    }
+
+    if (query.assignedTo) {
```

Comment:
This I suppose will be a dropdown box!

Reply:
Hey Karthikk, quick question: if it's a dropdown box and all the values are predetermined there, should I just use *=* here?
I assume that Frontend will load the user names into the dropdown, but will send their ID's to backend from the selected dropdown, is that correct?

Reply:
Exactly!